plugins/tiddlywiki/browser-storage/startup.js

```diff
@@ -53,6 +54,48 @@ exports.startup = function() {
 		$tw.wiki.addTiddler({title: ENABLED_TITLE, text: "no"});
 		$tw.browserStorage.clearLocalStorage();
 	});
+	// Helpers for protecting storage from eviction
+	var setPersistedState = function(state) {
+			$tw.wiki.addTiddler({title: PERSISTED_STATE_TITLE, text: state});
+		},
+		requestPersistence = function() {
+			setPersistedState("requested");
+			navigator.storage.persist().then(function(persisted) {
+				console.log("Request for persisted storage " + (persisted ? "granted" : "denied"));
+				setPersistedState(persisted ? "granted" : "denied");
+			});
+		},
+		persistPermissionRequested = false,
```

Comment:
I would prefer a `var` in front of all the variables and functions here. IMO it makes it easier to understand. ...

Reply:
Thanks @pmario. I thought I was following the convention I've seen in the other tiddlywiki code. All the variables seem to be initialized using a single `var`. Are you thinking this is a special case because there are multiple function definitions involved here?

Reply:
The core does indeed generally use a single `var` to declare multiple variables